### PR TITLE
C#: Start using 'options' files in tests

### PR DESCRIPTION
**Pull request by tamasvajk**
This PR moves all `semmle-extractor-options:` from `.cs` files to `options` files in the test folder. Except: 
- the ASP extractor also reads `options` files, and we can't mix C# options with ASP options, so `semmle-extractor-options` from `.cs` files where the folder also contains `.aspx` files are not moved.
- `--separate-compilation` is still file based, so these extractor options are not moved from `.cs` files.

### CODE REVIEW

File: csharp/ql/test/library-tests/aliases/Program.cs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-  semmle-extractor-options: /r:asm1=${testdir}/Assembly1.dll /r:asm2=${testdir}/Assembly2.dll
+
```
Comment (tamasvajk) [inline csharp/ql/test/library-tests/aliases/Program.cs:2]: TODO: This should be fixed up manually. Check for similar issues.

File: csharp/ql/test/library-tests/encoding/SBCS.cs [hunk 2]
```diff
@@ -1,6 +1,4 @@
-//  semmle-extractor-options: /codepage:1252
-
 class SBCS
 {
-    string sbcs = "�";
+    string sbcs = "�";
```
Comment (tamasvajk) [inline csharp/ql/test/library-tests/encoding/SBCS.cs:3]: This should be reverted.

**Comment by tamasvajk** [comment]
Tests are passing after the first commit. Only `csharp/ql/test/query-tests/Stubs/Minimal/MinimalStubsFromSource.expected` had to be updated.

**Review by hvitved — APPROVED** [review]
Excellent, thanks for making this change. Feel free to remove commits ffbfef6313c6cf3bd400393e2f999ebb0070550c and ab4b7b0abe4bf055ed8d8409c371eeceb78410ee from the history, if you want.

**Comment by tamasvajk** [comment]
@hvitved Thanks for the review. I rebased this PR, and removed the two commits.

**Review by hvitved — APPROVED** [review]

**Pull request merged**